package.json

```diff
@@ -5,17 +5,20 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "webpack-dev-server --open --mode development",
-    "build": "webpack --mode production"
+    "dev": "webpack-dev-server --open --mode development",
+    "start": "serve -s build",
```

Comment:
You seem to be changing our start command. This means we won't be able to run `yarn start` to run the app locally.

Reply:
yes, yarn start should be able to work locally, serve is For Heroku production deployment.